src/test/testFiles/valid/stackConstructor.manimdsl

```diff
@@ -0,0 +1,2 @@
+let x = new Stack<number>() {1, 2, 3};
```

Comment:
Just to confirm, this is now syntactically valid but we don't handle this in compiler backend yet right?

Reply:
No we do not but this should probs be another ticket